helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -693,15 +758,15 @@ public void handleDataChange(String dataPath, Object data) {
     try {
       updateNotificationTime(System.nanoTime());
       if (dataPath != null && dataPath.startsWith(_path)) {
-        NotificationContext changeContext = new NotificationContext(_manager);
-        changeContext.setType(NotificationContext.Type.CALLBACK);
-        changeContext.setPathChanged(dataPath);
-        changeContext.setChangeType(_changeType);
-        enqueueTask(changeContext);
-      }
-    } catch (Exception e) {
-      String msg =
-          "exception in handling data-change. path: " + dataPath + ", listener: " + _listener;
+      NotificationContext changeContext = new NotificationContext(_manager);
```

Comment:
format

Reply:
I go back and check your PR description. You did click the "Code Quality" bullet. Please ensure that when you click it, you really did so.